_episodes/08-connected-components.md

```diff
@@ -261,156 +319,388 @@ max: 11
 ~~~
 {: .output}
 
-The `dtype` of `labeled_image` is `int64`.
-This means that values in this image range from `-2 ** 63` to `2 ** 63 - 1`.
-Those are really big numbers.
-From this available space we only use the range from `0` to `11`.
-When showing this image in the viewer, it squeezes the complete range into 256 gray values.
-The range of our numbers does not produce any visible change.
-The `skimage` library has tools to cope with this a situation.
-In the `skimage.color` module has a function `label2rgb()` that will do the conversion.
-We have already used the `skimage.color` module to convert color images to gray scale images.
-`skimage.color.label2rgb()` will create a new color image.
-All objects are colored according to a list of colors that can be customized.
-In order to see our objects, we can add the following code to our program:
+The `dtype` of `labeled_image` is `int64`. This means that values in
+this image range from `-2 ** 63` to `2 ** 63 - 1`. Those are really
+big numbers.  From this available space we only use the range from `0`
+to `11`. When showing this image in the viewer, it squeezes the
+complete range into 256 gray values. Therefore, the range of our
+numbers does not produce any visible change.
+
+Fortunately, the skimage library has tools to cope with this
+situation. We can use the function `skimage.color.label2rgb()` to
+convert the colors in the image (recall that we already used the
+`simage.color.rgb2gray()` function to convert to grayscale). With
+`skimage.color.label2rgb()`, all objects are colored according to a
+list of colors that can be customized. We can use the following
+commands to convert and show the image:
 
 ~~~
 # convert the label image to color image
 colored_labeled_image = skimage.color.label2rgb(labeled_image, bg_label=0)
 
-# show the created image in the viewer
-skimage.io.imshow(colored_label_image)
+fig, ax = plt.subplots()
+plt.imshow(colored_label_image)
```

Comment:
```suggestion
plt.imshow(colored_labeled_image)
```
There's a typo here on the image name.

Reply:
Addressed in a82d659bb036937700f773606fb2e7b03f2e7a3a.